dist/raw/ProjectName/README.md

```diff
@@ -1,37 +1,52 @@
-# ProjectName
+# {{cookiecutter.project_slug}}
```

Comment:
@danielsousaio would changing the ProjectName to cookiecutter slug work here? I don't think so. Can you confirm?